ambari-metrics/ambari-metrics-host-monitoring/src/main/python/core/config_reader.py

```diff
@@ -83,9 +83,9 @@ def get_ca_certs_file_path(self):
 METRIC_FILE_PATH = configDefaults.get_metric_file_path()
 CA_CERTS_FILE_PATH = configDefaults.get_ca_certs_file_path()
 
-OUT_DIR = os.path.join(os.sep, "var", "log", "ambari-metrics-host-monitoring")
-SERVER_OUT_FILE = OUT_DIR + os.sep + "ambari-metrics-host-monitoring.out"
-SERVER_LOG_FILE = OUT_DIR + os.sep + "ambari-metrics-host-monitoring.log"
+OUT_DIR = os.path.join(os.sep, "var", "log", "ambari-metrics-monitor")
```

Comment:
Can this path be changed from the ini configuration file?

Reply:
It is hardcoded right now.